jobs/payment-jobs/tasks/ejv_payment_task.py

```diff
@@ -367,7 +365,7 @@ def _process_partial_refunds(cls,
                 invoice_number = f"#{pr_invoice.id}"
                 description = jv_params.disbursement_desc[: -len(invoice_number)] + invoice_number
                 description = f"{description[:100]:<100}"
-                flow_through = f"{pr_invoice.id:<110}-'PR'-{pr.id}"
+                flow_through = f"{f'{pr_invoice.id}-PR-{pr.id}':<110}"
```

Comment:
eg.
![image](https://github.com/user-attachments/assets/1b0f626c-316b-4b3b-bf60-4433e02b5db5)

Reply:
If you look at the disbursement job https://github.com/bcgov/sbc-pay/blob/main/jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

You'll see I took the data from invoices and partner disbursements and made it into one similar data model
that way the processing becomes quite a bit easier when generating the file.

Reply:
If you think of a partial refund, the only real difference for it, you're grabbing data from a different spot but it's the same idea just mapping a payment line items with an amount and a custom flow through field into an position EJV file

Reply:
cut a ticket: https://github.com/bcgov/entity/issues/28527